minetest/minetest
# Add package update detection on Content tab

Fixes #7327 and fixes #11799

Minetest now shows available updates for content without opening the ContentDB dialog. This uses the new updates API, which is quite fast to load (35ms + network latency, without cache)

![image](https://github.com/minetest/minetest/assets/2122943/0b847169-decb-4f73-9a3b-93fb6c0cfe70)
![image](https://github.com/minetest/minetest/assets/2122943/4dd1441f-9df7-4096-b718-2238302913fc)

## To do

This PR is Ready for Review

## How to test

* Install some content from the ContentDB dialog
* Edit the .conf file to reduce the release number
* Change tab to Start Game and Restart Minetest
* See a number on the content tab
* Change tab to Content and see packages with update icons
* Restart Minetest, see content tab. Number and icon should appear shortly
